From 8ee133cd06fdb087bf1d8634d0d393d0c50a6cb1 Mon Sep 17 00:00:00 2001
From: Nikhil Badola <nikhil.badola@freescale.com>
Date: Mon, 2 Jun 2014 16:50:27 +0530
Subject: [PATCH 087/124] drivers/usb : Remove compilation error for usb EHCI
 gadget

Remove compilation errors from USB EHCI gadget driver

Signed-off-by: Nikhil Badola <nikhil.badola@freescale.com>
Change-Id: Ie0ad80495ab16fe8537e544113dde806c790ff05
Reviewed-on: http://git.am.freescale.net:8181/21819
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Ramneek Mehresh <ramneek.mehresh@freescale.com>
Reviewed-by: Zhengxiong Jin <Jason.Jin@freescale.com>
---
 drivers/usb/gadget/udc/fsl_usb2_udc.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/usb/gadget/udc/fsl_usb2_udc.h b/drivers/usb/gadget/udc/fsl_usb2_udc.h
index 8471562..1300b393 100644
--- a/drivers/usb/gadget/udc/fsl_usb2_udc.h
+++ b/drivers/usb/gadget/udc/fsl_usb2_udc.h
@@ -20,6 +20,10 @@
 #define USB_MAX_CTRL_PAYLOAD		64
 #define USB_DR_SYS_OFFSET		0x400
 
+#ifdef CONFIG_SOC_LS1021A
+#undef CONFIG_ARCH_MXC
+#endif
+
  /* USB DR device mode registers (Little Endian) */
 struct usb_dr_device {
 	/* Capability register */
-- 
2.1.0.27.g96db324

